### PR TITLE
Welsh layout

### PR DESCRIPTION
**Pull request by tenextractor**

### CODE REVIEW

**Comment by tenextractor** [comment]
image

File: srcs/layouts/latn_qwerty_cy.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  </row>
+  <row>
+    <key key0="shift" key2="loc capslock"/>
+    <key key0="z"/>
+    <key key0="x" key2="loc †"/>
+    <key key0="c" key2="&lt;" key3="."/>
+    <key key0="v" key2="&gt;" key3=","/>
+    <key key0="b" key2="\?" key3="/"/>
+    <key key0="n" key2=":" key3=";"/>
+    <key key0="m" key2="&quot;" key3="'"/>
```
Comment (Julow) [inline srcs/layouts/latn_qwerty_cy.xml:35]: `'` (key3 here) is also in the center of the next key.

Comment (Julow) [reply srcs/layouts/latn_qwerty_cy.xml:35]: Shift and the delete key on the 3rd row are quite small. Couldn't the `'` be a swipe ?

Comment (tenextractor) [reply srcs/layouts/latn_qwerty_cy.xml:35]: ' is quite commonly used in Welsh, the shift and backspace are still as big as other keys.

Comment (Julow) [reply srcs/layouts/latn_qwerty_cy.xml:35]: Alright, then `'` should be removed from the corner of the `m` key. Making Shift and Backspace the same size as the other keys is fine but as they are more frequently used, having them bigger improves typing.

Comment (tenextractor) [reply srcs/layouts/latn_qwerty_cy.xml:35]: Ok, I will remove it from the `m` key.

Comment (tenextractor) [reply srcs/layouts/latn_qwerty_cy.xml]: Removed

File: srcs/layouts/latn_qwerty_cy.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <key key0="a" key1="loc tab" key2="`"/>
+    <key key0="s" key2="loc §" key3="loc ß"/>
+    <key key0="d"/>
+    <key key0="f"/>
+    <key key0="g" key2="-" key3="_"/>
+    <key key0="h" key2="=" key3="+"/>
+    <key key0="j" key4="}" key3="{"/>
+    <key key0="k" key3="[" key4="]"/>
+    <key key0="l" key2="|" key3="\\"/>
+    <key key0="accent_circonflexe" key2="accent_trema" key3="accent_grave" />
```
Comment (Julow) [inline srcs/layouts/latn_qwerty_cy.xml:25]: You have the dead key and the accented letters at the same time on the layout. This spot could be used for an other dead key or for '.

Comment (tenextractor) [reply srcs/layouts/latn_qwerty_cy.xml:25]: It's the same logic as the Hawaiian layout, the circumflex is the most common diacritic in Welsh and it just gives multiple options to the user to type it.

**Review by Julow — APPROVED** [review]
In this case, let's merge!

**Pull request merged**